.Rhistory

```diff
@@ -0,0 +1 @@
+usethis::use_mit_license()
```

Comment:
This looks like it was accidentally included.

Reply:
Most definitely